.dsh/tools/commit_push_refresh_pr/index.ts

```
@@ -216,70 +221,163 @@ export default async function commit_push_refresh_pr(input: {
+          plan,
+          notes: ["Publication did not establish the expected remote commit."],
+          resultJson: JSON.stringify({
+            ok: false,
+            step: "publication",
+            pullNumber: input.pullNumber,
+            headSha: pushResult.headSha,
+            remoteState: pushResult.remoteState,
+            blocker: pushResult.blocker,
+            remoteState: "expected-commit",
+            recovery:
+              "Re-read verification for the published commit, reconcile the PR branch, and continue evidence refresh only after every commit is verified. Do not create or push another commit.",
```

Biome (2.5.7)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)